### ISSUE TITLE
How to stop verbose message on error?

### ISSUE THREAD

**Issue by vipreshjha** [vivekkrbajpai/VowpalWabbit#1]
Currently when ever there is an error while connecting we ger console message like 
"err in vw :connect ECONNREFUSED host:port"

is there a way to stop this messages from being printed?

**Comment by vivekkrbajpai** [comment]
Hi,

Are you sure your VW is running on the same port you provided?
Above msg appear when your VW server is not running on the given port.

**Comment by vipreshjha** [comment]
Yes the VW service is running on the same port, but the error comes when some times the code is running on local boxes where service is is not running.
We just want to stop the error message on local & beta boxes.
